internal/command/console_interactive.go

```diff
@@ -57,18 +57,17 @@ func (c *ConsoleCommand) modeInteractive(session *repl.Session, ui cli.Ui) int {
 		// brackets we know not to execute the command just yet
 		fullCommand, openState := consoleState.UpdateState(line)
 
-		switch {
-		case openState > 0:
+		if openState > 0 {
 			// here there are open brackets somewhere, so we don't execute it
 			// as we are in a bracket we update the prompt. we use one . per layer pf brackets
 			l.SetPrompt(fmt.Sprintf("%s ", strings.Repeat(".", openState)))
-		default:
+		} else {
 			out, exit, diags := session.Handle(fullCommand)
 			if diags.HasErrors() {
 				c.showDiagnostics(diags)
 			}
 			if exit {
-				break
+				return 0
```

Comment:
Let's leave a break statement in there for convenience, it is not a problem since we changed `switch-case` to `if-else`